src/cpu/cpu-device.cpp

```
@@ -19,13 +21,17 @@ Result DeviceImpl::initialize(const DeviceDesc& desc, BackendImpl* backend)
-        m_info.timestampFrequency = 1000000000;
+        m_info.timestampFrequency = getCpuTimestampFrequency();
     }
 
     // Initialize features & capabilities
     addFeature(Feature::SoftwareDevice);
     addFeature(Feature::ParameterBlock);
     addFeature(Feature::TimestampQuery);
+    if (getCpuTimestampDomain() != CpuTimestampDomain::Unknown)
+    {
+        addFeature(Feature::TimestampCalibration);
+    }
```

Cppcheck (2.20.0)

[style] The function 'create' is never used.

(unusedFunction)